test/jdk/lib/client/ExtendedRobot.java

```diff
@@ -106,180 +103,8 @@ public ExtendedRobot(GraphicsDevice screen) throws AWTException {
-     *
-     *
-     * @param   buttons The button mask; a combination of one or more mouse button masks.
-     * @throws  IllegalArgumentException if the {@code buttons} mask contains the mask for
-     *          extra mouse button and support for extended mouse buttons is
-     *          {@link Toolkit#areExtraMouseButtonsEnabled() disabled} by Java
-     * @throws  IllegalArgumentException if the {@code buttons} mask contains the mask for
-     *          extra mouse button that does not exist on the mouse and support for extended
-     *          mouse buttons is {@link Toolkit#areExtraMouseButtonsEnabled() enabled}
-     *          by Java
-     *
-     * @see     #mousePress(int)
-     * @see     #mouseRelease(int)
-     * @see     InputEvent#getMaskForButton(int)
-     * @see     Toolkit#areExtraMouseButtonsEnabled()
-     * @see     java.awt.event.MouseEvent
-     */
-    public void click(int buttons) {
-        mousePress(buttons);
-        waitForIdle(DEFAULT_SPEED);
-        mouseRelease(buttons);
-        waitForIdle();
-    }
-
-    /**
-     * Clicks mouse button 1
-     *
-     * @throws  IllegalArgumentException if the {@code buttons} mask contains the mask for
-     *          extra mouse button and support for extended mouse buttons is
-     *          {@link Toolkit#areExtraMouseButtonsEnabled() disabled} by Java
-     * @throws  IllegalArgumentException if the {@code buttons} mask contains the mask for
-     *          extra mouse button that does not exist on the mouse and support for extended
-     *          mouse buttons is {@link Toolkit#areExtraMouseButtonsEnabled() enabled}
-     *          by Java
-     *
-     * @see     #click(int)
-     */
-    public void click() {
-        click(InputEvent.BUTTON1_DOWN_MASK);
-    }
-
-    /**
-     * Waits until all events currently on the event queue have been processed with given
-     * delay after syncing threads. It uses more advanced method of synchronizing threads
-     * unlike {@link java.awt.Robot#waitForIdle()}
-     *
-     * @param   delayValue  Additional delay length in milliseconds to wait until thread
-     *                      sync been completed
-     * @throws  sun.awt.SunToolkit.IllegalThreadException if called on the AWT event
-     *          dispatching thread
-     */
-    public synchronized void waitForIdle(int delayValue) {
-        super.waitForIdle();
-        delay(delayValue);
-    }
-
-    /**
-     * Waits until all events currently on the event queue have been processed with delay
-     * {@link #getSyncDelay()} after syncing threads. It uses more advanced method of
-     * synchronizing threads unlike {@link java.awt.Robot#waitForIdle()}
-     *
-     * @throws  sun.awt.SunToolkit.IllegalThreadException if called on the AWT event
-     *          dispatching thread
-     *
-     * @see     #waitForIdle(int)
-     */
-    @Override
-    public synchronized void waitForIdle() {
-        waitForIdle(syncDelay);
```

Comment:
This `waitForIdle(500)` is no longer called by tests(as it uses regular `java.awt.Robot#waitForIdle()`, so I assume you have verified that automated testing looks good. (I haven't gone through all the tests yet)